src/commands/collaboration-whitelist/index.js

```diff
@@ -1,27 +1,14 @@
 'use strict';
 
-const BoxCommand = require('../../box-command');
+const CollaborationAllowlistListCommand = require('../collaboration-allowlist');
 
-class CollaborationWhitelistListCommand extends BoxCommand {
-	async run() {
-		const { flags, args } = this.parse(CollaborationWhitelistListCommand);
-		let options = {};
+class CollaborationWhitelistListCommand extends CollaborationAllowlistListCommand {}
 
-		if (flags.fields) {
-			options.fields = flags.fields;
-		}
+['description', 'examples', '_endpoint', 'flags'].forEach((name) => {
+	CollaborationWhitelistListCommand[name] =
+		CollaborationAllowlistListCommand[name];
+});
 
-		let whitelistEntries = await this.client.collaborationWhitelist.getAllWhitelistedDomains(options);
-		await this.output(whitelistEntries);
-	}
-}
-
-CollaborationWhitelistListCommand.description = 'List collaboration whitelist entries';
-CollaborationWhitelistListCommand.examples = ['box collaboration-whitelist'];
-CollaborationWhitelistListCommand._endpoint = 'get_collaboration_whitelist_entries';
-
-CollaborationWhitelistListCommand.flags = {
-	...BoxCommand.flags
-};
+CollaborationWhitelistListCommand.hidden = true;
```

Comment:
Thanks @sujaygarlanka. I restored `box collaboration-whitelist` to be exactly like `box collaboration-allowlist` except it won't show in the list of commands as it is hidden.